packages/stack/src/TransitionConfigs/CardStyleInterpolators.tsx

```diff
@@ -63,6 +63,13 @@ export function forHorizontalIOS({
   };
 }
 
+export function forHorizontalIOSInverted(
+  props: StackCardInterpolationProps
+): StackCardInterpolatedStyle {
+  props.inverted = new Animated.Value(-1);
```

Comment:
Don't mutate objects. The mutation will alter the object in the parent too, not only here.